src/gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1554,34 +1554,34 @@ properties:
       - Gray
       - Green
       - Hazel
-      - Red & Violet
       - Other
+      - Red & Violet
```

Comment:
Would "Red & Violet" go above "Other"?

Reply:
We don't have 'Other' indicated as a non-informative terms such as 'Unknown', 'Not Reported' etc. but it does make sense that it isn't really informative. I can add it to the docs and update the TSV2YAML to sort it towards the bottom of enum lists.